pages.fr/common/install.md

```diff
@@ -0,0 +1,24 @@
+# install
+
+> Copie des fichiers et mets à jour leurs attributs.
+> Copie des fichiers (souvent des exécutables) dans un repertoire système comm `/usr/local/bin` et leur donne les permissions et propriétaires appropriés.
+
+- Copie des fichiers vers un destination:
+
+`install {{path/to/source}} {{path/to/destination}}`
+
+- Copie des fichiers vers une destination en mettant à jour leur propriétaire:
+
+`install -o {{user}} {{path/to/source}} {{path/to/destination}}`
+
+- Copie des fichiers vers une destination en mettant à jour leur groupe d'appartenance:
+
+`install -g {{user}} {{path/to/source}} {{path/to/destination}}`
+
+- Copie des fichiers vers une destination en mettant à jour leur mode:
+
+`install -m {{+x}} {{path/to/source}} {{path/to/destination}}`
+
+- Copie des fichier et met à jour leurs date d'accès et de modification à partir de leur source respective:
```

Comment:
```suggestion
- Copie des fichieres et met à jour leurs dates d'accès et de modification à partir de leurs sources respectives:
```

Reply:
I think we must keep the singular form here `leur date d'accès` because there is a single object per owner of the object

Reply:
You're right, second part is singular :+1: my bad